solr/core/src/test/org/apache/solr/cloud/CreateRoutedAliasTest.java

```diff
@@ -231,7 +217,106 @@ public void testV1() throws Exception {
     assertNotNull(meta);
     assertEquals("evt_dt", meta.get("router.field"));
     assertEquals("_default", meta.get("create-collection.collection.configName"));
-    assertEquals(null, meta.get("start"));
+    assertNull(meta.get("start"));
+  }
+
+  @Test
+  public void testUpdateRoudetedAliasDoesNotChangeCollectionList() throws Exception {
+
+    final String aliasName = getSaferTestName();
+    Instant start = Instant.now().truncatedTo(ChronoUnit.HOURS); // mostly make sure no millis
+    createTRAv1(aliasName, start);
+
+    String initialCollectionName =
+        TimeRoutedAlias.formatCollectionNameFromInstant(aliasName, start);
+    assertCollectionExists(initialCollectionName);
+
+    // Note that this is convenient for the test because it imples a different collection name, but
+    // doing this is an advanced operation, typically preceded by manual collection creations and
+    // manual
```

Comment:
the comment newline flow is bad.  Maybe use `/*` style; perhaps Spotless reflows that better?

Reply:
ah didn't catch this after I did spotless apply thx.